pkgs/tools/X11/devour/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, libX11 }:
```

Minor spacing thing:
```suggestion
{ stdenv, fetchFromGitHub, libX11 }:

```